rct229/rulesets/ashrae9012019/section16/section16rule2.py

```diff
@@ -0,0 +1,85 @@
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.ruleset_model_factory import produce_ruleset_model_instance
+from rct229.rulesets.ashrae9012019 import BASELINE_0, PROPOSED, USER
+from rct229.rulesets.ashrae9012019.ruleset_functions.compare_schedules import (
+    compare_schedules,
+)
+from rct229.utils.assertions import getattr_
+
+
+class Section16Rule2(RuleDefinitionListIndexedBase):
+    """Rule 2 of ASHRAE 90.1-2019 Appendix G Section 16 (Elevators)"""
+
+    def __init__(self):
+        super(Section16Rule2, self).__init__(
+            rmds_used=produce_ruleset_model_instance(
+                USER=False,
+                BASELINE_0=True,
+                PROPOSED=True,
+            ),
+            each_rule=Section16Rule2.BuildingRule(),
+            index_rmd="baseline",
+            id="16-2",
+            description="The baseline elevator motor use shall be modeled with the same schedule as the proposed design. Rule Assertion: B-RMD = P-RMD",
+            ruleset_section_title="Elevators",
+            standard_section="Section G3.1",
+            is_primary_rule=True,
+            list_path="ruleset_model_descriptions[0].buildings[*]",
```

Comment:
I believe list_path should be only "ruleset_model_descriptions[0]"

Reply:
Addressed.